### ISSUE TITLE
Lootcrate interaction does not work

### ISSUE THREAD

**Issue by TheTrueRusty** [ABravePanda/LootCrate#8]
Hey there,

Love the plugin btw, but as of a couple of days ago the interaction for players doesn't seem to work anymore. Might just be a silly space or something but I can't seem to find it. Players can see the crates, can vote but cannot use the granted key to unlock the crate. I updated to the latest version but my crates just dissapeared and the plugin got disabled, so I reverted to the second latest update and that works, however, the interaction still doesn't work.
![permissions](https://user-images.githubusercontent.com/16511382/109228088-a379e400-77c1-11eb-887d-0d562b5fa58f.png)

Thx
Rusty

**Comment by ABravePanda** [comment]
Feel free to add me on discord (nich#4476) if you have it as I can help more there. Thanks for letting me know!

**Comment by TheTrueRusty** [comment]
I've added you, even dm'ed you. Any idea as to when i'd get a reply? Members are kinda impatient...

**Comment by DanielLBarlow** [comment]
Sorry for the delay in response from Panda. I am one of his moderators on his discord. It sounds like you have discovered a bug but I unfortunately do not have the knowledge on how to fix said bug.
Panda has recently become slightly indesposed with some real world issues and has not been too active on discord. Even I struggle to get in contact with him haha.
We are very sorry for the delay and he will do his best to get back to you as soon as he is able to. Thank you for trying to be as patient as possible

**Comment by TheTrueRusty** [comment]
Thanks Daniel,

I fixed the issue by redoing the crate from scratch in the latest version. However, after a while it just bugs out again. I don't know what happened. I updated my other plugins to the latest version and thats when it started. Console spits out following error. 

![image](https://user-images.githubusercontent.com/16511382/111919494-9384b580-8a8a-11eb-88fa-61fcd0c2f7a4.png)

Any idea what it means? It hap^pened after updating to the newest version of the plugin. And the older version has a different way of reading the file I edit. 

Thanks in advance.
